src/renderer/pages/Staking/Overview/Overview.tsx

```diff
@@ -218,8 +241,10 @@ export const Overview = () => {
   const totalStakes = Object.values(staking).map((stake) => stake?.total || '0');
   const relaychainAsset = getRelaychainAsset(activeChain?.assets);
 
-  const toggleSelectedNominators = (address: Address) => {
-    if (selectedNominators.includes(address)) {
+  const toggleSelectedNominators = (address: Address, isAllSelected?: boolean) => {
+    const isSelected = isAllSelected === undefined ? selectedNominators.includes(address) : !isAllSelected;
```

Comment:
won't work in case if we want to unselect all shards (isAllSelected = false) but isSelected becomes true in case a shard selectedNominators.includes(address) will be true